Makefile

```diff
@@ -145,7 +150,8 @@ assets: $(GOBINDATA)
 build: ## Builds Thanos binary using `promu`.
 build: check-git deps $(PROMU)
 	@echo ">> building binaries $(GOBIN)"
-	@$(PROMU) build --prefix $(PREFIX)
+	@$(PROMU) build --prefix $(PREFIX)/.build/linux-amd64
+	@ln -sf $(PREFIX)/.build/linux-amd64/thanos thanos
```

Comment:
This is a bit confusing. I guess some other tools depend on `thanos` binary being in current directory?

Reply:
Yes, `scripts/genflagdocs` need it.